peekingduck/training/src/optimizers/schedules.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Scheduler Class to interact with Tensorflow
+
+    @staticmethod
+    def PiecewiseConstantDecay(parameters):
+        return tf.keras.optimizers.schedules.PiecewiseConstantDecay(**parameters)
+
+    @staticmethod
+    def ExponentialDecay(parameters):
+        return tf.keras.optimizers.schedules.ExponentialDecay(**parameters)
+
+    @staticmethod
+    def CosineDecay(parameters):
+        return tf.keras.optimizers.schedules.CosineDecay(**parameters)
+
+    @staticmethod
+    def InverseTimeDecay(parameters):
+        return tf.keras.optimizers.schedules.InverseTimeDecay(**parameters)
+
+    @staticmethod
+    def PolynomialDecay(parameters):
+        return tf.keras.optimizers.schedules.PolynomialDecay(**parameters)
+
+    @staticmethod
+    def CosineDecayRestarts(parameters):
+        return tf.keras.optimizers.schedules.CosineDecayRestarts(**parameters)
+
+"""
```

Comment:
Is the above comment supposed to help the developer use the different `Keras` optimizers? If yes, pls explicitly mention it in the comments (i.e. how to use). If not, then they should be removed.

Reply:
https://github.com/furyhawk/PeekingDuck/tree/47df8145fbba97081c6758cf66856b571d37db99

Removed comment.